mocks/jwt-decode.js

```diff
@@ -0,0 +1,7 @@
+export default (tokenAsDate) => {
```

Comment:
what is this mocking doing? A bit confused here. More comments...

Reply:
Ah, my bad. I should have done that already. Added more comments just now.